### ISSUE TITLE
Some typings are missing in published bundle

### ISSUE THREAD

**Issue by ktsn** [mysticatea/vue-eslint-parser#23]
I noticed the latest vue-eslint-parser (v2.0.2) bundle does not have some typings which the lib is using. If I use it in TypeScript with `--noImplicitAny` flag, it reports some errors like below:

```
node_modules/vue-eslint-parser/index.d.ts(55,34): error TS7016: Could not find a declaration file for module 'eslint-scope'. '**/node_modules/eslint-scope/lib/index.js' implicitly has an 'any' type.
node_modules/vue-eslint-parser/index.d.ts(554,33): error TS7016: Could not find a declaration file for module 'eslint-visitor-keys'. '**/node_modules/eslint-visitor-keys/lib/index.js' implicitly has an 'any' type.
```

I just created a minimal reproduction repo: https://github.com/ktsn/vue-eslint-parser-test

This looks because vue-eslint-parser has own typings for external deps and does not publish them. Maybe, it would be better to make a PR to [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped) to publish these typings?

**Comment by mysticatea** [comment]
Oh, thank you for this report.

I had overlooked that errors.
I will make PRs to DefinitelyTyped.

**Comment by mysticatea** [comment]
Please track those PRs. Thank you!

- https://github.com/DefinitelyTyped/DefinitelyTyped/pull/23728
- https://github.com/DefinitelyTyped/DefinitelyTyped/pull/23729

**Comment by ktsn** [comment]
Thank you!
